File: change_color.php
<?php

$directories = [
    __DIR__ . '/resources/views/public',
    __DIR__ . '/resources/views/layouts'
];

foreach ($directories as $dir) {
    if (is_dir($dir)) {
        $files = glob($dir . '/*.blade.php');
        foreach ($files as $file) {
            $content = file_get_contents($file);
            $content = str_replace('blue-', 'primary-', $content);
            $content = str_replace('indigo-', 'primary-', $content);
            $content = str_replace('purple-', 'primary-', $content);
            $content = str_replace('sky-', 'primary-', $content);
            // Replace specific gradients from-[hex] to-[hex] 
            $content = str_replace("from-['#20a306']", 'from-primary-600', $content); // if any
            file_put_contents($file, $content);
        }
    }
}
echo "Colors updated successfully!";

File: change_color.py
import os

directories = [
    r"d:\CODE\WEB_SMK\resources\views\public",
    r"d:\CODE\WEB_SMK\resources\views\layouts"
]

for directory in directories:
    for filename in os.listdir(directory):
        if filename.endswith(".blade.php"):
            filepath = os.path.join(directory, filename)
            with open(filepath, 'r', encoding='utf-8') as f:
                content = f.read()
            
            # Replace colors
            content = content.replace('blue-', 'primary-')
            content = content.replace('indigo-', 'primary-')
            content = content.replace('purple-', 'primary-')
            content = content.replace('sky-', 'primary-')
            
            with open(filepath, 'w', encoding='utf-8') as f:
                f.write(content)

print("Colors updated successfully in Blade files.")
